inc/integrations/class-pico.php

```diff
@@ -64,6 +64,17 @@ public function setup() {
 		}
 
 		add_filter( 'wp_irving_verify_coral_user', [ $this, 'verify_pico_user_for_sso' ] );
+
+		// Use staging API urls.
+		if ( get_option( 'irving_integrations' )['pico']['use_staging'] ?? false ) {
```

Comment:
If `irving_integrations` option doesn't exist, this will throw a PHP warning because of trying to access array indexes in something that is not an array.  `get_option` takes a default as a second param... setting the default to an array should solve this.

Reply:
I confirmed that the null coalescing doesn't mind,

<img width="540" alt="Screen Shot 2020-09-15 at 6 23 20 PM" src="https://user-images.githubusercontent.com/665107/93271231-c454fc80-f780-11ea-9be5-2768c1b0b448.png">